auth.go

```diff
@@ -273,6 +282,10 @@ func NewCentralFromConfig(config *Config) (central *Central, err error) {
 		panic(err)
 	}
 
+	if userStore, err = NewUserStoreDB_SQL(&config.UserStore.DB); err != nil {
+		panic(err)
```

Comment:
Follow the convention here for NewPermitDB_SQL, which wraps up the error returned by NewPermitDB_SQL in a friendlier message.